### PR TITLE
Refactor login

### PR DESCRIPTION
**Pull request by Samikshap85**
###Description
>This PR adds refactoring of login feature where the user register first and then only logs in

### CODE REVIEW

File: Features1/tests/config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { chromium } = require("playwright");
 const clearDatabases = require("./databaseCleaner");
 
 BeforeAll(async () => {
-  global.browser = await chromium.launch({});
+  global.browser = await chromium.launch({ headless: false, slowMo: 100 });
```
Comment (kiranadh1452) [inline Features1/tests/config.js]:
```suggestion
  global.browser = await chromium.launch();
```

File: Features1/tests/features/login.feature [hunk 2]
```diff
@@ -4,8 +4,14 @@ Feature:
...
-    Given the user has navigated to the login page
-    When  the user enters "samikshapaudel111@gmail.com" and "Hello123"
+     Given the user has navigated to the register page
+     And the user has been register with following user information
+        |name    |email         |password|confirmpassword|
+        |demo    |demo@gmail.com|hello123|hello123       |
+    And the home page has been displayed on the webUI
+    And user has logged out 
+    And the user has navigated to the login page   
+    When  the user enters "demo@gmail.com" and "hello123"
```
Comment (kiranadh1452) [inline Features1/tests/features/login.feature]: These lines are differently indented.

File: Features1/tests/stepDefinitions/loginContext.js [hunk 3]
```diff
@@ -36,3 +36,8 @@ Then(
     await page.pause();
   }
 );
+
+Given('the has entered following contact information', function (dataTable) {
+  
+});
+
```
Comment (kiranadh1452) [inline Features1/tests/stepDefinitions/loginContext.js]: Why empty?

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features1/tests/stepDefinitions/loginContext.js [hunk 4]
```diff
@@ -36,3 +36,4 @@ Then(
     await page.pause();
```
Comment (kiranadh1452) [inline Features1/tests/stepDefinitions/loginContext.js:35]: Why page.pause() ?

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features1/tests/features/login.feature [hunk 5]
```diff
@@ -4,8 +4,14 @@ Feature:
     So that i can access to the home page
 
 Scenario: Login with valid credential
-    Given the user has navigated to the login page
-    When  the user enters "samikshapaudel111@gmail.com" and "Hello123"
+    Given the user has navigated to the register page
+    And the user has been register with following user information
```
Comment (SagarGi) [inline Features1/tests/features/login.feature]:
```suggestion
    And the user has been registered with following user information
```

File: Features1/tests/features/login.feature [hunk 6]
```diff
@@ -4,8 +4,14 @@ Feature:
...
+        |name    |email         |password|confirmpassword|
+        |demo    |demo@gmail.com|hello123|hello123       |
+    And the home page has been displayed on the webUI
+    And user has logged out 
+    And the user has navigated to the login page   
+    When  the user enters "demo@gmail.com" and "hello123"
```
Comment (SagarGi) [inline Features1/tests/features/login.feature]:
```suggestion
    When the user login with email "demo@gmail.com" and password "hello123"
```

File: Features1/tests/config.js (hunk 1 above)
Comment (SagarGi) [inline Features1/tests/config.js]:
```suggestion
  global.browser = await chromium.launch({});
```

Comment (SagarGi) [reply Features1/tests/config.js]: remove this then we can merge this.

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**